modules/timepoint_list/README.md

```diff
@@ -0,0 +1,48 @@
+# Timepoint List
+
+## Purpose
+
+The `timepoint_list` module serves as an entry-point for data entry
+about a candidate. It serves as a dashboard for a candidate linking
+to other modules that provide specific functionality about a candidate.
+
+## Intended Users
+
+The `timepoint_list` module is primarily intended for data entry
+staff who are accessing a candidate.
+
+## Scope
+
+The `timepoint_list` module provides a mechanism to link to
+information about candidates. 
+
+NOT in scope:
+
+The `timepoint_list` module does *not* provide any data entry
+functionality or mechanisms to configure study timepoints,
+only access timepoints of specific candidates.
+
+## Permissions
+
+The `timepoint_list` module requires the user accessing it to
+have access to at least 1  timepoint of the candidate being
+accessed. This can either occur by having the `access_all_profiles`
+permission, being a member of the same site as the candidate's
+RegistrationCenterID, or having access to a CenterID in common
+with one of the candidate's timepoints.
+
```

Comment:
useEDC.  -> will show in top bar of basic candidate profile

Reply:
I checked the code and there's no reference to useEDC in the module.